### PR TITLE
Add get release version from RPC

### PR DESCRIPTION
**Pull request by luqz**

### CODE REVIEW

**Comment by kaikai1024** [comment]
can you work on this branch?
https://github.com/cryptape/cita-common/pull/229

I will grant you permission.

**Comment by kaikai1024** [comment]
bors try

**Comment by kaikai1024** [comment]
Sorry. Still working on the circleci problem.
weird, both you and @ouwenkg can not work.
So, we will use travisCI temporarily。

**Comment by kaikai1024** [comment]
You can drop tht `test` commit now. @luqz

**Comment by kaikai1024** [comment]
And you can run the ci on your own repo if there are many pull requests. that's will faster to check the result

**Review by leeyr338 — DISMISSED** [review]

**Review by kaikai1024 — CHANGES_REQUESTED** [review]
please drop the test commit

**Comment by luqz** [comment]
> please drop the test commit

OK

**Review by leeyr338 — DISMISSED** [review]

File: jsonrpc-types/src/rpctypes/software_version.rs [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// CITA
+// Copyright 2016-2017 Cryptape Technologies LLC.
```
Comment (kaikai1024) [inline jsonrpc-types/src/rpctypes/software_version.rs]: 2016-2019

Comment (luqz) [reply jsonrpc-types/src/rpctypes/software_version.rs]: ok

File: jsonrpc-types/src/request/tests/params.rs [hunk 2]
```diff
@@ -227,4 +227,6 @@ fn serialize_and_deserialize() {
         ["earliest"],
         (BlockNumber::earliest())
     );
+
+    test_ser_and_de!(GetVersionParams, [], ());
```
Comment (kaikai1024) [inline jsonrpc-types/src/request/tests/params.rs:231]: What's the test used for?

Comment (luqz) [reply jsonrpc-types/src/request/tests/params.rs]: just test serialization, like log.rs in the same dir.

Comment (luqz) [reply jsonrpc-types/src/request/tests/params.rs:231]: All params have this test, like test_ser_and_de!(BlockNumberParams, [], ()).

**Comment by leeyr338** [comment]
Plz do not using `push -f` every time. 
You need to have a new commit while you are changing the code, so I can know what you have changed for every commit.
Otherwise, I must review the code from begin to end time after time.

**Comment by kaikai1024** [comment]
> Plz do not using `push -f` every time.
> You need to have a new commit while you are changing the code, so I can know what you have changed for every commit.
> Otherwise, I must review the code from begin to end time after time.

That's the right operation. you can click the `force-push`, and can read the changes

**Comment by kaikai1024** [comment]
> You need to have a new commit while you are changing the code

That's not a good choice for me.

**Review by kaikai1024 — APPROVED** [review]

**Review by leeyr338 — APPROVED** [review]

**Comment by leeyr338** [comment]
> > Plz do not using `push -f` every time.
> > You need to have a new commit while you are changing the code, so I can know what you have changed for every commit.
> > Otherwise, I must review the code from begin to end time after time.
> 
> That's the right operation. you can click the `force-push`, and can read the changes

That sounds OK.

**Pull request merged**